src/components/vm/consoles/vnc.tsx

```diff
@@ -404,22 +480,23 @@ export class VncActive extends React.Component {
             this.observer.disconnect();
     }
 
-    onDisconnected(detail) { // server disconnected
-        console.info('Connection lost: ', detail);
-        this.props.state.setConnected(false);
+    onDisconnected(clean: boolean) { // server disconnected
+        console.info('Connection lost: clean ', clean);
+        this.props.state.setDisconnected(clean);
     }
 
-    onInitFailed(detail) {
+    onInitFailed(detail: unknown) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)